workflow/scripts/prioritization/variants.py

```
@@ -8,179 +8,175 @@
+                            continue
+
+                    # resolved alleles specific to vep
+                    alleles_vep = self.resolve_alleles(entry)
+
+                    chrom = entry.CHROM
+                    start = entry.affected_start
+                    stop = entry.affected_end
+                    ref = entry.REF
+                    alts = entry.ALT
+
+                    for alt_i, alt_v in enumerate(alts):
```

Ruff (0.15.13)

[warning] Loop control variable `alt_i` not used within loop body

(B007)